lib/temporal/metadata/workflow.rb

```diff
@@ -3,15 +3,18 @@
 module Temporal
   module Metadata
     class Workflow < Base
-      attr_reader :namespace, :id, :name, :run_id, :attempt, :headers
+      attr_reader :namespace, :id, :name, :run_id, :attempt, :task_queue, :headers, :memo, :run_started_at
 
-      def initialize(namespace:, id:, name:, run_id:, attempt:, headers: {})
+      def initialize(namespace:, id:, name:, run_id:, attempt:, task_queue:, headers: {}, memo:, run_started_at:)
```

Comment:
nit: assuming headers and memo are very similar, so both should either have a default value or not

Reply:
Checked all call sites, removing the default value for both